### DISCUSSION TITLE
How to config vpnhood docker

### DISCUSSION THREAD

**Discussion by Evolve6996** [vpnhood/VpnHood#111] [Q&A] [answered]
hello 
I installed vpnhood as a docker container how can I get access keys and config it? 

cause i didn't found appsetting.json inside var/lib/docker/containers/vpnhood container id

thanks

**Comment by trudyhood** [comment, marked as answer]
Hi,
VpnHood Docker is pretty new. I've created a document for that. Please check it out and let me know if you need more information.
https://github.com/vpnhood/VpnHood/wiki/How-to-run-VpnHood-Server-on-Linux-using-docker

**Comment by Evolve6996** [comment]
thanks, 👍  I was able to run the server. 😄 

I have two other questions:

1- I stop and start the server with this command docker start/stop (container-id) but it won't give me info like when I was running it as a regular Linux package for example it won't say the server start on this port or stuff like that. is there any way to see that info ?

2-there is a fake hostname parameter when you run the server what is the purpose of that?

**Reply by trudyhood** [reply]
> _1- I stop and start the server with this command docker start/stop (container-id) but it won't give me info like when I was running it as a regular Linux package for example, it won't say the server start on this port or stuff like that. is there any way to see that info ?_

You can see the log file or omit -d options or use the following docker command
docker logs <container id>

> _2-there is a fake hostname parameter when you run the server what is the purpose of that?_

Sorry, I couldn't understand about fake hostname! But VpnHood creates a self-signed certificate to establish HTTPS connection. You need a hostname for HTTPS connection, so VpnHood creates a random hostname and assigns it to a self-signed certificate. You can use your own self-signed certificate or a valid one, but it won't increase security. VpnHood client can validate the server's authenticity by comparing the thumbprint of the certificate with the one that exists in the access token.
